### PR TITLE
Create script to get user locations

### PR DESCRIPTION
**Pull request by jphechter**
## Description

Create a script that can be used to populate user location information in BFS Users Table

### Ticket

[185396010](https://www.pivotaltracker.com/story/show/185396010)

### Approach

The script added in this PR scans DynamoDB and prints all users' email and provided location to a CSV file. This is sensitive information, so the output file is included in `.gitignore`.

Running this script requires AWS session keys which can be acquired through the AWS login console. Additionally, you must provide the name of the table you want to query.

I was running into issues with `ProvisionedThroughputExceededException`. Tried a couple different things, but the best solution I was able to come up with short of modifying our DynamoDB instance was to just add a pause in the script. Every 10000 records, the script pauses for 5 seconds before automatically continuing.

## Code author checklist

- [X] I have performed a self-review of my code
- [X] I have created and/or updated relevant documentation, if necessary
- [X] I have not used any relative imports
- [X] I have checked for and removed instances of unused config from CMS
- [X] I have pruned any instances of unused code
- [X] I have not added any markdown to labels, titles and button text in config

### CODE REVIEW

File: scripts/get-user-location.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+fi
+
+aws_command="aws dynamodb scan"
+output_file="get-user-location-output.csv" # This filename is included in .gitignore to prevent accidental commits
+max_items=1000
+last_evaluated_key=""
+counter=0
+
+read -p "Enter the AWS region: " region
+read -p "Enter the table name: " table_name
```
Comment (jphechter) [inline scripts/get-user-location.sh:15]: This intentionally obfuscated.

Comment (adidner) [inline scripts/get-user-location.sh:15]: [dust] Might be cool to have environment variables in our .env that this draws from by default? That way we don't type to type if the usual suspect here is almost always the same 

something like "hit enter to default to your env vars or type in the AWS region"

**Review by adidner — APPROVED** [review]
Looks good to me, (although I am no expert in AWS or shell scripting)

File: scripts/get-user-location.sh (hunk 1 above)
Comment (jphechter) [reply scripts/get-user-location.sh:15]: Good thought. These are in the `web/.env` file. I'll see about pulling from there.

**Pull request merged**